src/routes/Landing.tsx

```diff
@@ -0,0 +1,338 @@
+              margin={["auto", null, null, null, null, 0]}
+            >
+              <Text as="h2" fontSize={[6, null, 7, 8]} fontWeight="bold">
+                How to use Retro tool
+              </Text>
+              <Text fontSize={[4, null, 5]} color="grey" my={4}>
+                The main and only screen is a 4 columns layout where you can add
+                different topics and action items (unless you are on tablet or
+                mobile, in that case the UI will adapt).
+              </Text>
+              <Text fontSize={[4, null, 5]} color="grey" mb={[8, null, 10]}>
+                A retro consists of 4 simple guided steps, you can always see
+                the phase you're in the breadcrumb and you can advance to the
+                next step with the button placed in the header.
+              </Text>
+
+              <RetroSteps>Brainstorm</RetroSteps>
+              <Text textStyle="landingBase" mt={1} mb={8}>
+                The first screen is where{" "}
+                <strong>you and your team can add topics</strong> like{" "}
+                <em>what went well</em>, <em>what can be improved</em> and{" "}
+                <em>questions</em>, usually this step is done asynchronously. A
+                good idea is to share the retro url some days before the retro
+                meeting.
+              </Text>
+
+              <RetroSteps>Group &amp; vote</RetroSteps>
+              <Text textStyle="landingBase" mt={1} mb={8}>
+                This step is to <strong>group and vote</strong> so you know what
+                the most important topics are for your team. Usually this step
+                and the following are done synchronously with your team.
+              </Text>
+
+              <RetroSteps>Add action items</RetroSteps>
+              <Text textStyle="landingBase" mt={1} mb={8}>
+                This step is where your team{" "}
+                <strong>discusses the prioritized topics</strong> and where you
+                can add action items to address those topics after the retro.
+              </Text>
+
+              <RetroSteps>Done</RetroSteps>
+              <Text textStyle="landingBase" mt={1}>
+                The last step is just to finish a retro, and at this point you
+                can <strong>export all the topics</strong> in plain text or{" "}
+                <strong>create a linked retro</strong>.
+              </Text>
+              <Text textStyle="landingBase" mt={4}>
+                <strong>Tip:</strong> You can create a retro with a{" "}
+                <strong>custom name</strong> just writing it in the url, and if
+                you choose this pattern{" "}
+                <em>https://retrotool.app/any-name-here-1</em>, when you create
+                a new retro in the last step it will create the retro{" "}
+                <em>https://retrotool.app/my-awesome-retro-2</em> and it will
+                show all the previous action items).
```

Comment:
```suggestion
                <strong>custom name</strong> by writing it in the url. If
                you choose this pattern{" "}
                <em>https://retrotool.app/any-name-here-1</em>, when you create
                a linked retro in the last step it will create the retro{" "}
                <em>https://retrotool.app/my-awesome-retro-2</em> and it will
                show all the previous action items).
```